HerPublicWebsite/Views/StaticPages/SessionExpired.cshtml

```diff
@@ -0,0 +1,20 @@
+@using HerPublicWebsite.Controllers
+@{
+    ViewBag.Title = "Session expired";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <h1 class="govuk-heading-xl">
+            Your answers have been deleted
+        </h1>
+        <div class="govuk-body">
+            <p>We have deleted your answers because you did not do anything for 10 minutes.
```

Comment:
10 minutes matches the session expiry value in `Startup.cs` - we should remember to change it if that is ever updated.

Reply:
Could you please add a comment to the relevant line in `Startup.cs` to say this?
Also is there any guidance on session length on GOV.UK? 10 minutes seems quite short, I'd suggest bumping it up to 30 minutes.

Reply:
The DWP example has 20 minutes, RTV looks like 1 hour, so 30 mins seems like a decent compromise. I can ask Kay at the same time as the copy question.